smithy-typescript-codegen/src/main/resources/software/amazon/smithy/typescript/codegen/base-package.json

```diff
@@ -3,23 +3,36 @@
   "description": "${packageDescription}",
   "version": "${packageVersion}",
   "scripts": {
-    "clean": "npm run remove-definitions && npm run remove-maps && npm run remove-js",
+    "clean": "npm run remove-definitions && npm run remove-dist && npm run remove-js && npm run remove-maps",
     "build-documentation": "npm run clean && typedoc ./",
-    "prepublishOnly": "tsc",
+    "prepublishOnly": "yarn build",
     "pretest": "tsc",
-    "remove-definitions": "rimraf *.d.ts && rimraf ./commands/*.d.ts && rimraf ./types/*.d.ts",
+    "remove-definitions": "rimraf ./types",
+    "remove-dist": "rimraf ./dist",
     "remove-documentation": "rimraf ./docs",
-    "remove-js": "rimraf *.js && rimraf ./commands/*.js && rimraf ./shared/*.js && rimraf ./types/*.js",
-    "remove-maps": "rimraf *.js.map && rimraf ./commands/*.js.map && rimraf ./shared/*.js.map && rimraf ./types/*.js.map",
+    "remove-js": "rimraf *.js && rimraf ./commands/*.js && rimraf ./lib/*.js && rimraf ./models/*.js && rimraf ./protocols/*.js",
```

Comment:
./lib was removed in https://github.com/awslabs/smithy-typescript/pull/51.  those get pulled in from @aws-sdk/smithy-client